apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.utils.ts

```diff
@@ -123,18 +123,20 @@ export const generateUpdateColumnPayload = (
   const isOriginallyPrimaryKey = primaryKeyColumns.includes(originalColumn.name)
 
   // Only append the properties which are getting updated
+  const name = field.name.trim()
   const type = field.isArray ? `${field.format}[]` : field.format
-  const comment = (field.comment?.length ?? '') === 0 ? null : field.comment
+  const comment = ((field.comment?.length ?? '') === 0 ? null : field.comment)?.trim()
+  const check = field.check?.trim()
 
   const payload: Partial<UpdateColumnPayload> = {}
-  if (!isEqual(originalColumn.name, field.name)) {
-    payload.name = field.name
+  if (!isEqual(originalColumn.name, name)) {
```

Comment:
So we don't accidentally rename columns that already contain a space is it worth adding a `trim()` here?
```suggestion
  if (!isEqual(originalColumn.name.trim(), name)) {
```

Reply:
hmm very fair point - otherwise might bring down their app
i'll add a comment as well